db/migrate/20170117064747_add_awarded_at_and_draft_exp_to_course_experience_points_records.rb

```diff
@@ -0,0 +1,29 @@
+class AddAwardedAtAndDraftExpToCourseExperiencePointsRecords < ActiveRecord::Migration
+  def change
+    add_column :course_experience_points_records, :draft_points_awarded, :integer
+    add_column :course_experience_points_records, :awarded_at, :datetime
+    add_column :course_experience_points_records, :awarder_id, :integer,
+               foreign_key: { references: :users }
+
+    Course::Assessment::Submission.joins(:experience_points_record).
+      where(workflow_state: ['attempting', 'submitted', 'graded']).find_each do |submission|
```

Comment:
With my local dev with 20k submissions in total, around 120s. I think production currently has around 30k submissions. 

For readability we can keep this for now, chatted with @allenwq and it should be ok.